mobile/README.md

```diff
@@ -2,6 +2,18 @@
 
 ## Prerequisites
 
+###Go language
+
+- Install go: `brew install go`
```

Comment:
Should we instead link to https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#preliminaries-for-installing-from-source to make it a bit more generic and not MacOS specific?